src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -2651,12 +2651,6 @@ private void CheckForRequiredMemberAttribute(BindingDiagnosticBag diagnostics)
                     diagnostics.Add(ErrorCode.ERR_RequiredMembersBaseTypeInvalid, method.GetFirstLocation(), BaseTypeNoUseSiteDiagnostics);
                 }
             }
-
-            if (this.IsExtension)
-            {
-                _ = Binder.GetWellKnownTypeMember(DeclaringCompilation, WellKnownMember.System_Runtime_CompilerServices_CompilerFeatureRequiredAttribute__ctor, diagnostics, GetFirstLocation());
```

Comment:
>Binder.GetWellKnownTypeMember(DeclaringCompilation, WellKnownMember.System_Runtime_CompilerServices_CompilerFeatureRequiredAttribute__ctor, diagnostics, GetFirstLocation()); [](http://example.com/codeflow?start=20&length=173)

Is this the check that enables extension usage on Desktop framework? I am asking because the changes that I have seen so far didn't look like target framework related.

Reply:
Yes, that attribute doesn't exist on Desktop (and we don't synthesize a definition for it) so was blocking tests